docs/Ways-of-Working/Governance/Problem-Governance/Challenge-Mapping.md

```diff
@@ -1,39 +1,76 @@
 
 ## Triggers
+Typical triggers for RAID include (but are not limited to):
 
-Challenges should be documented as soon as they are known, throughout the problem lifecycle.
+- **Requirement Elicitation** - as requirements are gathered for the problem during the discovery zone 
+- **Design** - during the design zone and design workshops, triggers could include architectural decisions that introduce technical risks, assumptions about third party integrations, or dependencies on specific design components
+- **Resource Allocation** - changes in resource availability or allocation
+- **Technologies and/or Tools** - selecting new technologies or tools for the solution 
+- **Change Requests** - whenever there are changes to solution requirements, scope, or timeline
+- **Testing and Quality Assurance** - issues identified during testing and quality assurance activities can trigger RAID items relating to defects, performance issues, or dependencies on external systems 
 
-The delivery team must identify appropriate times to collectively review and update their challenge map, so that it is kept fresh and relevant, and can be effectively used for planning.
 
-## Considerations
+## Communication Plan
+
+Delivery teams are responsible for ensuring relevant stakeholders and/or colleagues are informed about the status of key RAID items and any actions being taken that could potentially impact delivery, other delivery teams' plans or stakeholders and other departments.
+
+RAID communication plans typically include and document:
 
-A well established technique for challenge mapping is a [RAID Log](../../Toolkit/RAID.md).
+1. **Communication Channels** - with **delivery teams:** via regular team meetings or RAID planning events or **with stakeholders:** via [delivery status updates and reports](../Governance/Delivery-Governance/Delivery-Status-Reporting/index.md), ad-hoc communications to address urgent or critical RAID items, 
+2. **Content & Format** - RAID item summary, detailed descriptions, action items and trends in RAID dat over time
+3. **Escalation Process** - defined escalation process for addressing high-priority or critical RAID items that require immediate attention or resolution 
+4. **Feedback Mechanism** - encourage feedback from stakeholders and team members on the effectiveness of RAID communication and any improvements or suggested adjustments required
+5. **Review & Continuous Improvement** - regularly review the effectiveness of the RAID communication plan, gathering feedback from stakeholders and team members and identify opportunities for improvement
 
-!!! toolkit "Toolkit"
+!!! Note
+    Not all RAID items need to be communicated to all stakeholders, and in some situations can confuse rather than inform a stakeholder. Reporting on RAID should be tailored for the appropriate audiences, focussing on the key items of most concern or perceived impact.
```

Comment:
Would it be possible to have an example around this? The work we do as engineers can be very technical and we can find it difficult. Sometimes something which we think is really important is something that stakeholders don't really understand or care about.

Reply:
Its really difficult to provide an example because the approach you take will vary depending on who your stakeholders are and quite often a group of stakeholders will include those who are really interested in the technical stuff and those who have no understanding of what is being discussed.  In a way it can be a bit of trial and error to find the right level so I'd be happy to take a look at any items you aren't sure about to figure out how best (or whether) they should be communicated to stakeholders.